lib/internal/streams/BufferList.js

```diff
@@ -73,4 +75,9 @@ module.exports = class BufferList {
     }
     return ret;
   }
+
+  [customInspectSymbol]() {
```

Comment:
I’d just use `util.inspect.custom` here, it saves people looking at this code one round-trip to another file

Reply:
Done